src/Symfony/Component/Config/Definition/BaseNode.php

```diff
@@ -74,16 +74,16 @@ public static function setPlaceholder(string $placeholder, array $values): void
     }
 
     /**
-     * Sets a common prefix for dynamic placeholder values.
+     * Adds a common prefix for dynamic placeholder values.
      *
      * Matching configuration values will be skipped from being processed and are returned as is, thus preserving the
      * placeholder. An exact match provided by {@see setPlaceholder()} might take precedence.
      *
      * @internal
      */
-    public static function setPlaceholderUniquePrefix(string $prefix): void
+    public static function addPlaceholderUniquePrefix(string $prefix): void
```

Comment:
even if it's internal, we should keep the name to break a BC break between the DI and the Config components (see deps=low/high failures)